src/controleur/AdherentCompteController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class AdherentCompteController extends Controller
+{
+
+    // Classes PDO utilisées
+
+    // Variables utilisées par la vue
+    private Adherent $adherent;
+
+    // Variables utilisées par le controller
+    private string    $requestMethod;
+
+    public function __construct(string $requestMethod)
+    {
+        $this->getAdherent();
+
+        $this->requestMethod = $requestMethod;
+        $this->titre    = 'Mon Compte';
+        $this->view     = 'adherentCompte';
+
+        if ($this->requestMethod === 'POST') {
+            self::handlePosts();
+        }
+
+        $this->data = array(
+            'titre'      => $this->titre,
+            'adherent'   => $this->adherent
+        );
+
+        $this->render($this->view, $this->data);
+    }
+
+    private function getAdherent()
+    {
+        if (isset($_SESSION['current_adherent'])) {
+            $this->adherent = $_SESSION['current_adherent'];
+        } else {
+            Router::redirect('accueil');
+        }
+    }
+
+    private static function handlePosts()
```

Comment:
Du coup si tu ajoutes l'adherentBd au début, il faut le passer en argument.
```suggestion
    private static function handlePosts(AdherentBD adherentBD)
```